CMakeLists.txt

```
@@ -362,7 +362,7 @@ IF (WIN32)
     # Fix static library dependencies
     set(SDL_MIXER_EXTRA_LIBS ssp)    
     set(SDL_EXTRA_LIBS mingw32 dxerr8)
-    set(SDL_IMG_EXTRA_LIBS jbig deflate)
+    set(SDL_IMG_EXTRA_LIBS jbig deflate lerc)
```

The library archive is called `libLerc.dll.a`. so try
```suggestion
    set(SDL_IMG_EXTRA_LIBS jbig deflate Lerc)
```